submit/controllers/process.py

```diff
@@ -304,10 +305,20 @@ def file_preview(params, session: Session, submission_id: int, token: str,
                  **kwargs) -> Response:
     submitter, client = user_and_client_from_session(session)
     submission, submission_events = load_submission(submission_id)
-    prod = Compiler.get_product(submission.source_content.identifier,
-                                submission.source_content.checksum, token)
-    headers = {'Content-Type': prod.content_type}
-    return prod.stream, status.OK, headers
+    if (submission.source_content.source_format == SubmissionContent.Format.PDF):
+        subfiles = FileManager.get_upload_status(
+            submission.source_content.identifier,
+            token)
+        pdf_name = next(( file.name for file in subfiles.files if file.file_type=='PDF'))
```

Comment:
@DavidLFielding We might want to think about a route that exposes the ``get_single_file`` method on the upload workspace, so that clients don't have to do this.

Reply:
With `https:\\asYetUndefinedPdfService.arxiv.org/submissions/1234/getPreviewPdf` client's would not need to do this.

Reply:
Sure, but we don't have that right now. What you have here is fine for the purpose of this PR. The objective of "having one place where the submission PDF lives" is on the docket, and at that point we can gleefully revise this